server/src/models/socobouser.ts

```diff
@@ -0,0 +1,5 @@
+export class SocoboUser {
```

Comment:
I would suggest to not use the Socobo-Prefix but I think thats the fault of postgres, since User is a protected key word or so!?